src/TLuaInterpreter.cpp

```diff
@@ -16963,25 +16963,29 @@ void TLuaInterpreter::initLuaGlobals()
     lua_register(pGlobalLua, "getMudletInfo", TLuaInterpreter::getMudletInfo);
     // PLACEMARKER: End of main Lua interpreter functions registration
 
-    const auto separator = QDir::separator();
     const auto nativeHomeDirectory = mudlet::getMudletPath(mudlet::profileHomePath, hostName);
-
-    luaL_dostring(pGlobalLua, QStringLiteral("package.path = [[%1%2?.lua;%1%2?%2init.lua;]] .. package.path").arg(nativeHomeDirectory, separator).toUtf8().constData());
-
-    luaL_dostring(pGlobalLua, QStringLiteral("package.cpath = [[%1%2?;]] .. package.cpath").arg(nativeHomeDirectory, separator).toUtf8().constData());
+#if defined(Q_OS_WIN32)
+    luaL_dostring(pGlobalLua, QStringLiteral("package.path = toNativeSeparators([[%1/?.lua;%1/?/init.lua;]]) .. package.path").arg(nativeHomeDirectory).toUtf8().constData());
+    luaL_dostring(pGlobalLua, QStringLiteral("package.cpath = toNativeSeparators([[%1/?.dll;]]) .. package.cpath").arg(nativeHomeDirectory).toUtf8().constData());
+#else
+    luaL_dostring(pGlobalLua, QStringLiteral("package.path = [[%1/?.lua;%1/?/init.lua;]] .. package.path").arg(nativeHomeDirectory).toUtf8().constData());
+    luaL_dostring(pGlobalLua, QStringLiteral("package.cpath = [[%1/?.so;]] .. package.cpath").arg(nativeHomeDirectory).toUtf8().constData());
+#endif
 
 #if defined(Q_OS_LINUX)
-    // if using LuaJIT, adjust the cpath to look in /usr/lib as well - it doesn't by default
```

Comment:
Note the removal of the LuaJIT stuff following a recent, brief discussion we had about it on Discord!